docs/rules/valid-jsdoc.md

```diff
@@ -341,6 +342,25 @@ function add(num1, num2) {
 }
 ```
 
+### requireParamType
+
+If you use https://www.npmjs.com/package/esdoc-flow-plugin, make a copy for all param types not needed.
```

Comment:
If you use flowtype and esdoc + esdoc-flow-plugin or documentation.js, documentation of the types will be automatic, thus no need to specify parameter types manually.